_includes/code/web-example.html

```diff
@@ -0,0 +1,80 @@
+<!-- 
+  provide the following variables:
+  - id: unique id for the example
+  - html: html code snippet
+  - js: js code snippet
+  - scripts: a commad separate list of required scripts. Allowed values are pbjs,gpt,astjs . default is pbjs,ast
```

Comment:
With the `scripts` include an example can specify the scripts required for this example. I chose a sensible default that most examples use